readme.txt

```diff
@@ -3,6 +3,7 @@
 = [TBD] TBD =
 
 * Release - B23.Goldorak
+* Fix - Adding Configuration feature, to enable simple feature flag and other checks, with less boilerplate. See [readme](https://github.com/the-events-calendar/tribe-common/pull/1923/files#diff-cf03646ad083f81f8ec80bbdd775d8ac45c75c7bc1bf302f6fb06dfa34a1dc64) for more details. [ECP-1505]
```

Comment:
Lets split this into two Changelogs.

Reply:
@bordoni not sure I follow - what do you mean two changelogs? Can you give me an example of what you mean?